testing_using_container.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -x
+set -eu
+
+# https://build.opensuse.org/package/show/systemsmanagement:Agama:Staging/agama-testing
+CIMAGE=registry.opensuse.org/systemsmanagement/agama/staging/containers/opensuse/agama-testing:latest
+# rename this if you test multiple things
+CNAME=agama
+
+test -f service/agama.gemspec || echo "You should run this from a checkout of agama"
+
+# destroy the previous instance, can fail if there is no previous instance
+podman stop ${CNAME?} || : no problem if there was nothing to stop
+podman rm ${CNAME?} || : no problem if there was nothing to remove
+
+# Update our image
+podman pull ${CIMAGE?}
+
+# Optionally use a git version of ruby-dbus
+# because Agama pushes its limits so it's better
+# to set up for easy testing
+MORE_VOLUMES=()
+if [ "${WITH_RUBY_DBUS-}" = 1 ]; then
```

Comment:
Maybe document that env variable at the top of script together with some documentation what script does?